Control coverage/Feature/ServiceFabric.md

```diff
@@ -1,7 +1,391 @@
+
+<!-- - **PowerShell** 
+
+	 ```powershell 
+	 $variable = 'apple' 
+	 ```  
+
+- **Enforcement Policy** 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+-->
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to get certificate details of Service Fabric resource: - /subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/clusters?api-version=2018-02-01 
+ <br />
+**Properties:** properties.certificate
+ <br />
+
+<br />
+
+___ 
+
+## Azure_ServiceFabric_AuthN_Client_AuthN_AAD_Only 
+
+### DisplayName 
+Use Azure Active directory for client authentication on Service Fabric clusters 
+
+### Rationale 
+Using the native enterprise directory for authentication ensures that there is a built-in high level of assurance in the user identity established for subsequent access control. All Enterprise subscriptions are automatically associated with their enterprise directory (xxx.onmicrosoft.com) and users in the native directory are trusted for authentication to enterprise subscriptions. 
+
+### Control Spec 
+
+> **Passed:** 
+> AAD authentication is enabled.
+> 
+> **Failed:** 
+> AAD authentication is not enabled.
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 A Service Fabric cluster offers several entry points to its management functionality, including the web-based Service Fabric Explorer, Visual Studio and PowerShell. Access to the cluster must be controlled using AAD. Refer: https://docs.microsoft.com/en-in/azure/service-fabric/service-fabric-cluster-creation-setup-aad 
+
+<!-- - **PowerShell** 
+
+	 ```powershell 
+	 $variable = 'apple' 
+	 ```  
+
+- **Enforcement Policy** 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+
+-->
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to get Azure Active Directory details and its related property: - /subscriptions/{subscriptionId}/providers/Microsoft.ServiceFabric/clusters?api-version=2018-02-01  
+ <br />
```

Comment:
The Properties part is not coming as bold.
You need to shift the < br /> tag to the previous line. Refer other PR for changes.

- API description < br Tag>
**Properties:**